sopel/cli/utils.py

```diff
@@ -266,7 +278,8 @@ def load_settings(options):
     elif 'SOPEL_CONFIG' in os.environ:
         name = os.environ['SOPEL_CONFIG'] or name  # use default if empty
 
-    filename = find_config(config.DEFAULT_HOMEDIR, name)
+    filename = find_config(
+        getattr(options, 'configdir', config.DEFAULT_HOMEDIR), name)
```

Comment:
Why is this using `getattr`? `options.configdir` has a default value, defined just a few dozen lines earlier.

Reply:
Good call!